net-ftp/lftp/lftp-4.6.5.ebuild

```diff
@@ -32,9 +30,10 @@ RDEPEND="
 		>=net-proxy/dante-1.1.12
 		virtual/pam
 	)
+	gnutls? ( >=net-libs/gnutls-1.2.3 )
 	ssl? (
-		gnutls? ( >=net-libs/gnutls-1.2.3 )
-		openssl? ( dev-libs/openssl:0 )
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
 	)
```

Comment:
Moving gnutls outside of ssl is a bad idea.   As I said above, you're changing the meaning of these USE flags.  What you want is something like the following
ssl? (
   gnutls? ( net-libs/gnutls )
   !gnutls? (
        !libressl? ( dev-libs/openssl:0= )
        libressl? ( dev-libs/libressl:0= )
    )
)

So, if you pick USE=ssl, you'll get some ssl.  If you further pick gnutls then you get gnutls, and if you pick libressl you get that.  If you pick neither, then you fall back on openssl.  Also, the REQUIRED_USE will change.

BTW if you got this from the Gentoo Libressl Project page, its wrong and I'll fix it now.  Sorry I think I made that mistake